packages/@react-spectrum/list/src/ListViewItem.tsx

```diff
@@ -226,7 +226,7 @@ export function ListViewItem<T>(props: ListViewItemProps<T>) {
               }
             </div>
           }
-          {isDropTarget && !dropIndicator?.dropIndicatorProps['aria-hidden'] &&
+          {isListDroppable && !dropIndicator?.isHidden &&
```

Comment:
Digging again into why isDropTarget doesn't work for Android, would like if we could only render a single drop indicator at a time

Reply:
There are several issues here:
1. The detected modality is incorrect for Android Talkback, `isVirtualClick` doesn't return the correct modality (will need more investigation, seems like the way we check for Android Talkback double tap in usePress doesn't solve this either.)
2. If we don't render all the drop indicators when a drag session starts, DragManager doesn't have a good way to "unhide" other valid drop targets when the user swipes to move the screenreader cursor. Only the initial valid drop target is visible to the screen reader since it is the only row that `isDropTarget` is visible for.

I'm thinking the current approach is best for now, additional investigation can be done as a followup